README.md

```diff
@@ -1,23 +1,70 @@
-<h1 align="center">Jellyfin ApiClient.js</h1>
+<h1 align="center">Jellyfin API Client for JavaScript</h1>
 <h3 align="center">Part of the <a href="https://jellyfin.media">Jellyfin Project</a></h3>
 
+---
+
 <p align="center">
-Jellyfin ApiClient.js is a JavaScript library for interacting with Jellyfin's REST API.
+<img alt="Logo Banner" src="https://raw.githubusercontent.com/jellyfin/jellyfin-ux/master/branding/SVG/banner-logo-solid.svg?sanitize=true"/>
+<br/>
+<br/>
+<a href="https://github.com/jellyfin/jellyfin-apiclient-javascript">
+<img alt="MIT License" src="https://img.shields.io/github/license/jellyfin/jellyfin-apiclient-javascript.svg"/>
```

Comment:
Might actually be better to just write `License` here so we don't have to update it for every repository. It will take a while before the change bubbles out to other projects but we can start here.

Reply:
I think having the license spelled out would be better if Shield.io eventually goes out and for accessibility.

We don't change these often, so we may as well keep it there, I think